chef/site-cookbooks/wca/recipes/default.rb

```diff
@@ -103,7 +103,7 @@
   action :create_if_missing
 end
 
-node.default['brightbox-ruby']['version'] = "2.3"
+node.default['brightbox-ruby']['version'] = "2.4"
```

Comment:
Just checking if I understand: currently "2.4" isn't used anywhere else, but you're suggest pulling "2.4" out into a variable to make the code a little more select documenting?